local-interchain/cmd/local-ic/chains.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+)
+
+type chains struct {
+	Configs []string `json:"chain_configs"`
+}
+
+var chainsCmd = &cobra.Command{
+	Use:   "chains [config.json]",
+	Short: "List all current chains or outputs a current config information",
+	Args:  cobra.RangeArgs(0, 1),
+	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return GetFiles(), cobra.ShellCompDirectiveNoFileComp
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		chainsDir := path.Join(GetDirectory(), "chains")
+
+		if len(args) == 0 {
+			configs := chains{GetFiles()}
+
+			bz, _ := json.MarshalIndent(configs, "", "  ")
+			fmt.Printf("%s\n", bz)
+		} else {
+			config := args[0]
+			filePath := path.Join(chainsDir, config)
```

Comment:
Adding more context: If a user was able to specify a direct path, there is no way to know which relayer or where to output logs (without a bunch of ENV variables or CLI args & lengthy commands)

If a developer needed to store files in multiple locations, having `ICTEST_HOME` with the `chains` dir and `configs` would be fine. Could add some env variable like ICTEST_CHAINS=chain_dir or something like that? 

I do not foresee developers wanting this though since ICTEST_HOME makes it possible to do CI testing with it already